docs/root/development/performance/device_conditions.rst

```diff
@@ -0,0 +1,80 @@
+.. _dev_device_conditions:
```

Comment:
```suggestion
.. _dev_performance_device_conditions:
```

Reply:
I can update this in my PR (going to open after this one lands)